### ISSUE TITLE
prefer_inline, false positives

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1529]
```abap
FORM bar.
  DATA: ls_files TYPE string,
        ls_like LIKE ls_files.
  ls_files = 'abc'.
ENDFORM.
```

```abap
    DATA BEGIN OF ls_udmo_long_text.
    DATA language TYPE dm40t-sprache.
    DATA header   TYPE thead.
    DATA content TYPE xstring.
    DATA END OF ls_udmo_long_text.
```
